src/commands/bindle.rs

```diff
@@ -123,19 +123,12 @@ impl Prepare {
             .app
             .as_deref()
             .unwrap_or_else(|| DEFAULT_MANIFEST_FILE.as_ref());
-        let source_dir = crate::app_dir(app_file)?;
 
         let dest_dir = &self.staging_dir;
 
-        let (invoice, sources) = spin_publish::expand_manifest(app_file, self.buildinfo, &dest_dir)
+        let bindle_id = spin_publish::prepare_bindle(app_file, self.buildinfo, dest_dir)
             .await
-            .with_context(|| format!("Failed to expand '{}' to a bindle", app_file.display()))?;
-
-        let bindle_id = &invoice.bindle.id;
-
-        spin_publish::write(&source_dir, &dest_dir, &invoice, &sources)
-            .await
-            .with_context(|| crate::write_failed_msg(bindle_id, dest_dir))?;
+            .with_context(|| crate::write_failed_msg(dest_dir))?;
```

Comment:
Will this result in the "write failed" message even if it was the expand that failed?  Just a bit anxious that this could point people in the wrong direction if so.

Reply:
Missed this during rebase, thanks!